### PR TITLE
PlayerInfo:Dolby:Amlogic: Trace compile fix

### PR DESCRIPTION
**Pull request by HaseenaSainul**

### CODE REVIEW

File: PlayerInfo/Dolby/AMLogic/Implementation.cpp [hunk 1]
```diff
@@ -23,6 +23,7 @@
 #include <string.h>
 #include "hardware/audio.h"
 
+using namespace WPEFramework;
```
Comment (MFransen69) [inline PlayerInfo/Dolby/AMLogic/Implementation.cpp:26]: Hi Haseena, if I may ask why did you add this? Was there something else not compiling? For the change below it would not be needed it I'm right

Comment (HaseenaSainul) [reply PlayerInfo/Dolby/AMLogic/Implementation.cpp:26]: Yes it is required to compile the Trace log (Trace Category is under WPEFramework namespace)

**Review by modeveci — DISMISSED** [review]
Worked on yocto build

**Comment by pwielders** [comment]
Could we also make this a dedicated fix, e.g. i.s.o Trace::Information -> WPEFramework::Trace::Information. or the otherway around, the line just above this trace, remove the WPEFramework namespace before the ERROR_<CODe>.
Currently (for @modeveci  to learn dutch saysing) "hinkt op twee gedachtes" (it is using both possible ways, and declering it global, which if possible, should be avoided if it is not used a lot, and some statement are fully qualified)

**Comment by HaseenaSainul** [comment]
Updated as per @pwielders comments

File: PlayerInfo/Dolby/AMLogic/Implementation.cpp (hunk 1 above)
Comment (MFransen69) [reply PlayerInfo/Dolby/AMLogic/Implementation.cpp]: Eh, but now you removed it, so it was not needed then?

Comment (modeveci) [reply PlayerInfo/Dolby/AMLogic/Implementation.cpp]: "Hinkt op twee gedachtes" :)

Comment (HaseenaSainul) [reply PlayerInfo/Dolby/AMLogic/Implementation.cpp]: yea, instead used it directly from the Trace API WPEFramework::Trace::Information

**Review by modeveci — APPROVED** [review]

**Review by pwielders — APPROVED** [review]
